TemporalConvolution.lua

```diff
@@ -14,7 +16,8 @@ function TemporalConvolution:__init(inputFrameSize, outputFrameSize,
     local nOutputPlane = outputFrameSize
     self.inputFrameSize = inputFrameSize
     self.outputFrameSize = outputFrameSize
-    cudnn.SpatialConvolution.__init(self, nInputPlane, nOutputPlane, kW, kH, 1, dH,0,padH)
+    self.nDim = 4
```

Comment:
What's the 4th dimension here? I can only think of 3 dimensions that TemporalConvolution could have: {minibatch, number of features, sequence}
If VolumetricConvolution has nDim == 5, and SpatialConvolution has nDim == 4, then you would expect this to be nDim == 3, anyway.